app/Http/Controllers/Organization/OrganizationController.php

```diff
@@ -132,9 +131,7 @@ public function viewRecommendations($id)
     {
         if(auth()->guard('organization')->id() == $id)
```

Comment:
This  condition will go within the service. In case of failure the service should return null. In this function, you will check if the returned value is null or not to make the redirections